src/middlewared/middlewared/plugins/smb.py

```diff
@@ -99,6 +99,14 @@ def key_cp(encoding):
             ] + initial
         }
 
+    @private
+    async def interface_choices(self):
```

Comment:
If this is to be used in smb.update we should name it `bindip_choices` and make it public, with a docstring.